client-frameworks-support/client-support-angular/projects/client-support-angular/README.md

```diff
@@ -70,6 +70,7 @@ In your Angular route configuration, you can add in any of the following preload
 
  ```javascript
 {path: 'luigi-client-support-preload',component: Sample1Component,data: { fromVirtualTreeRoot: true }}
+{path: 'luigi-client-support-preload',component: Sample1Component,data: { fromVirtualTreeRoot: : {"truncate": "*/projects"} }}
```

Comment:
maybe a second example without the asterisk would be good, also for the following occurences